src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -23,6 +23,11 @@ const getDisplayableFilters = (filterDefinitions: FilterDefinitions) => {
   return result
 }
 
+const getDefaultDisplayFilters = (filterDefinitions: FilterDefinitions) =>
+  Object.entries(filterDefinitions)
+    .filter(([filterName, filterValue]) => filterValue.default)
+    .map(([filterName, filterValue]) => filterName)
```

Comment:
```suggestion
    .filter(([_, filterValue]) => filterValue.default)
    .map(([filterName, _]) => filterName)
```
For the unused fields on each entry